src/test/Components/CreateImageWizard/CreateImageWizard.content.test.js

```diff
@@ -111,7 +111,9 @@ describe('Step Packages', () => {
       ));
 
       // select aws as upload destination
-      await user.click(await screen.findByTestId('upload-aws'));
+      await waitFor(
+        async () => await user.click(await screen.findByTestId('upload-aws'))
+      );
```

Comment:
Oooh interesting, so the thing is, the test wasn't working before I added this because there's a Spiner now while the targets are loading. And it made the test crash because the element wasn't immediately available. Lemme double check what I did just to make sure it makes sense :slightly_smiling_face:

Reply:
Ok, I went a bit too hard replacing all of them. Some needs to be kept though. I'll do some cleaning. Thanks!!

Reply:
@regexowl I cleaned just now all the ones I could without breaking the testsuite. Is it better in your opinion?

Reply:
@lavocatt It is, thank you!

Reply:
Cool! thanks, I'll resolve all the related issues.